docs/concept-exercises.md

```diff
@@ -93,9 +93,68 @@ See the C# floating-point-numbers exercise's [design.md file][csharp-docs-design
 
 ### `.meta/config.json`
 
-This file contains meta information on the exercise, which currently includes the exercise's contributors, and (optionally) language version requirements.
-
-See the C# floating-point-numbers exercise's [config.json file][csharp-docs-config.json] for an example.
+This file contains meta information on the exercise, which may include
+
+- the exercise's contributors
+- the exercise's authors
+- which exercise it was forked from
+- language version requirements
+
+#### Example
+
+Assume that the user FSharpForever has written an exercise called `basics` for the F# track. PythonProfessor adapts the exercise for the Python track. Later on, the user PythonPerfection improves the exercise.
+
+##### `.../python/.../basics/.meta/config.json`
+
+```json
+{
+    "contributors": [
+        {
+            "github_username": "PythonProfessor",
+            "exercism_username": "PythonProfessor"
+        },
+        {
+            "github_username": "PythonPerfection",
+            "exercism_username": "PythonPerfection"
+        }
+    ],
+    "authors": [
```

Comment:
Maybe I misread this, but I would have expected the author here to be `PythonProfessor`, not the original author. `PythonProfessor` is the author of this particular exercise after all. If this interpretation is correct, `PythonProfessor` can be removed from the above contributors list I think. @iHiD @SleeplessByte what do you think?

Reply:
Wasn't this the reason to have two keys in the first place? Being able to credit who wrote the narrative or who wrote the original exercise, even if they didn't contribute directly to this particular implementation? Now that I think about it, maybe the `"forked_from"` key is sufficient to get that information.

Reply:
Yeah. I agree.

And the original author will get credits implcitiely via the forked option.

Reply:
Agreed.

Reply:
As Sacha and I cross-posted, for clarity, I'm saying that I think this should have `PythonProfessor` as the explicit exercise author, and that the original author would get credits in the system via the `forked_from` key. I think we're all agreeing.